client-app/core/composables/useAuth.ts

```diff
@@ -4,6 +4,7 @@ import { useFetch } from "@/core/api/common";
 import { errorHandler, toServerError } from "@/core/api/common/utils";
 import { globals } from "@/core/globals";
 import { TabsType, unauthorizedErrorEvent, useBroadcast } from "@/shared/broadcast";
+import { useWebPushNotifications } from "@/shared/push-messages/composables/useWebPushNotifications";
```

Comment:
useAuth is the base level service
useWebPushNotifications is an optional service

try not to make base level depends on optional

Reply:
we have broadcast service
also think about other tabs